linode/manager
# fix: [M3-6937] - Hide Wazuh Marketplace app

## Description 📝
There's a deployment issue with the Wazuh app that's causing support tickets to be created. @tbaka requested to temporarily hide this app until it's fixed

## How to test 🧪
Go to `/linodes/create?type=One-Click` and ensure that the Wazuh app is no longer visible
